src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -14,30 +14,35 @@ end
 
 """
     SubcellLimiterIDP(equations::AbstractEquations, basis;
-                      local_minmax_variables_cons = String[],
+                      local_twosided_variables_cons = String[],
                       positivity_variables_cons = String[],
                       positivity_variables_nonlinear = [],
                       positivity_correction_factor = 0.1,
-                      spec_entropy = false,
-                      math_entropy = false,
+                      local_onesided_variables_nonlinear = [],
                       max_iterations_newton = 10,
                       newton_tolerances = (1.0e-12, 1.0e-14),
                       gamma_constant_newton = 2 * ndims(equations))
 
 Subcell invariant domain preserving (IDP) limiting used with [`VolumeIntegralSubcellLimiting`](@ref)
 including:
-- Local maximum/minimum Zalesak-type limiting for conservative variables (`local_minmax_variables_cons`)
+- Local two-sided Zalesak-type limiting for conservative variables (`local_twosided_variables_cons`)
 - Positivity limiting for conservative variables (`positivity_variables_cons`) and nonlinear variables
 (`positivity_variables_nonlinear`)
-- One-sided limiting for specific and mathematical entropy (`spec_entropy`, `math_entropy`)
+- Local one-sided limiting for nonlinear variables, e.g. `entropy_spec` and `entropy_math`
+with `local_onesided_variables_nonlinear`
 
-Conservative variables to be limited are passed as a vector of strings, e.g. `local_minmax_variables_cons = ["rho"]`
-and `positivity_variables_cons = ["rho"]`. For nonlinear variables the specific functions are
-passed in a vector, e.g. `positivity_variables_nonlinear = [pressure]`.
+To use these three limiting options use the following structure:
+
+***Conservative variables*** to be limited are passed as a vector of strings, e.g.
+`local_twosided_variables_cons = ["rho"]` and `positivity_variables_cons = ["rho"]`.
+For ***nonlinear variables***, the wanted variable functions are passed within a vector: To ensure
+positivity use a plain vector including the desired variables, e.g. `positivity_variables_nonlinear = [pressure]`.
+For local one-sided limiting pass the variable function combined with the requested bound
+(`min` or `max`) as a tuple, e.g. `local_onesided_variables_nonlinear = [(Trixi.entropy_spec, min)]`.
```

Comment:
I suggest to add this, such that it's clearer what `min`/`max`means:
```suggestion
For local one-sided limiting pass the variable function combined with the requested bound
(`min` or `max`) as a tuple. For instance, to impose a lower local bound on the specific entropy 
use `local_onesided_variables_nonlinear = [(Trixi.entropy_spec, min)]`.
```